crypto/bio/bss_mem.c

```diff
@@ -53,6 +53,13 @@ static const BIO_METHOD secmem_method = {
     NULL,                      /* mem_callback_ctrl */
 };
 
+struct buf_mem_dgram_st {
+    /* Pointer into the buffer for where the dgram starts */
```

Comment:
Why not comment to the right of the field like the existing code?

Reply:
I don't feel this is particularly necessary. But done anyway.